client/src/main/java/org/evosuite/testcase/DefaultTestCase.java

```diff
@@ -75,7 +75,7 @@ public class DefaultTestCase implements TestCase, Serializable {
     /**
      * Coverage goals this test covers
      */
-    private transient Set<TestFitnessFunction> coveredGoals = new LinkedHashSet<>();
+    private final Set<TestFitnessFunction> coveredGoals = new LinkedHashSet<>();
```

Comment:
Why does not the `coveredGoals` need to be `transient`?

Reply:
"TestRedundancy.java" (i.e., one of the implemented test smells) uses the method "getCoveredGoals()" of "DefaultTestCase.java":
`Set<TestFitnessFunction> testGoals = testChromosome.getTestCase().getCoveredGoals();`

We were having problems because "testGoals" was null. In turn, this was happening because "coveredGoals" (a field in "DefaultTestCase.java") was transient, hence the change.